external/model-preparation-algorithm/mpa_tasks/utils/data_utils.py

```diff
@@ -17,7 +17,8 @@ def get_cls_img_indices(labels, dataset):
     for i, item in enumerate(dataset):
         item_labels = item.annotation_scene.get_labels()
         for i_l in item_labels:
-            img_indices[i_l.name].append(i)
+            if i_l in labels:
```

Comment:
I'm actually still wondering if this is the right way to go. is the structure of label_schema a problem or is it the problem to be solved here?

Reply:
Label schema is incoming properly, but previous code does not check the domain of dataset_item. So, if we train detection -> classification task, detection label is coming as item_labels on above code, this caused KeyError. 
To resolve this, I adopted simple if statement to check the domain of item_labels based on label schema (because it comes properly)